### ISSUE TITLE
[Feature Request] Normalize projection matrix

### ISSUE THREAD

**Issue by edgarriba** [kornia/kornia#463]
Would great to have the following normalization homography function

```python
def normalize_projection_matrix(H: torch.Tensor, eps: float = 1e-6) ->torch.Tensor:
    scale_val: torch.Tensor = H[..., -1:, -1:]
    return H / (scale_val + eps)
```

not sure about to `.detach()` the scale value.

**Comment by edgarriba** [comment]
can be done through `normalize_transformation` 
https://github.com/kornia/kornia/blob/df93618db8bb393d2eedcdee2c39b8a80eba8cc7/kornia/geometry/epipolar/fundamental.py#L49
